core/01_core.js

```diff
@@ -26,8 +26,11 @@
     ObjectAssign,
     SymbolFor,
   } = window.__bootstrap.primordials;
-  const ops = window.Deno.core.ops;
-  const opIds = Object.keys(ops).reduce((a, v, i) => ({ ...a, [v]: i }), {});
+  const core_ = window.Deno.core;
```

Comment:
Looks like the changes to 01_core.js are not necessary (since the objs object itself is cleared rather than recreated).